lib/plugins.js

```diff
@@ -4,15 +4,39 @@
 
 'use strict'
 
+var resolveFrom = require('resolve-from')
+
 /**
  * @method plugins
  *
  * @param {Object} config PostCSS Config
  *
  * @return {Array} plugins PostCSS Plugins
  */
+
+function requirePlugin (plugin, cwd) {
+  var err = false
+
+  try {
+    return require(plugin)
+  } catch (_err) {
+    err = _err
+  }
+
+  try {
+    return require(resolveFrom(cwd, plugin))
+  } catch (_err) {}
+
```

Comment:
it seems to me [flatry](https://github.com/ymatuhin/flatry) will be appropriate here

Reply:
Didn't want to add an extra dependency ;-)